platform_versioned_docs/version-23.4.0/launch/advanced.mdx

```diff
@@ -56,10 +56,10 @@ Enable this option to ensure that Nextflow pulls the latest version from the Git
 
 ## Main script
 
-Nextflow will attempt to run the script named `main.nf` in the project repository by default. You can change this with the `manifest.mainScript` option or by providing the script filename to run in this field.
+Nextflow will attempt to run the script named `main.nf` in the project repository by default. A custom script filename can be configured using `manifest.mainScript` or by providing the script filename in this field.
```

Comment:
```suggestion
Nextflow will attempt to run the script named `main.nf` in the project repository by default. You can configure a custom script filename in `manifest.mainScript` or you can provide the script filename in this field.
```

Reply:
I've changed the wording to have fewer gerunds and also lean more toward active voice. I think this will be something we need to look out for in general as the docs are definitely a bit inconsistent in this regard. Let me know if this doesn't sit well with you in this context.

Reply:
It sits well with me and makes a lot of sense. Thanks!